RockWeb/Blocks/Finance/TransactionEntry.ascx.cs

```diff
@@ -291,6 +299,10 @@ protected override void OnInit( EventArgs e )
             lSuccessHeader.Text = GetAttributeValue( "SuccessHeader" ).ResolveMergeFields( configValues );
             lSuccessFooter.Text = GetAttributeValue( "SuccessFooter" ).ResolveMergeFields( configValues );
 
+            // Determine account campus context mode
+            _accountCampusContext = GetAttributeValue( "AccountCampusContext" ).AsType<int>();
+            _currentCampusContext = RockPage.GetCurrentContext( EntityTypeCache.Read( typeof( Campus ) ) ).Id;
```

Comment:
You're correct.  I didn't realize that I needed to enable the option to for `Clear Selection Text` in the Campus Context Setter to properly test that.